package.json

```diff
@@ -82,6 +82,7 @@
     "@livelybone/copy": "^2.5.4",
     "@livelybone/date-generator": "^3.2.1",
     "@livelybone/vue-popper": "^2.3.1",
+    "moment": "^2.24.0",
```

Comment:
I think use moment is not a good idea because it makes the bundle size of app much larger

Reply:
I understand you prefer to keep the bundle small, so you did not used moment. On the other side, the configurable date format is an important feature for a datepicker, moment is widely spread, and it would take much longer to implement without it

Reply:
I agree with the option that the configurable date format is an important feature for a datepicker, but use the moment inner the component will make it much restricted. Maybe provide a `formatter` prop is a better choice?

Reply:
moment is used to format and also to parse. the component will be less intuitive and moment will be used outside...

Reply:
I mean, provide the `formatter` prop for user to custom the formatting action，so the user can choose what ever date library he want，not only moment, also like date-fns, date-format, dayjs... and so on. You can not force the user to use moment in a program that already use date-fns or the others

Reply:
Removing the dependency will cost more props : a parser and validator are also required

Reply:
What do the parser and validator do? You worry about the date string emitted out of the component maybe was wrong?